<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminUI</class>
 <widget class="QMainWindow" name="AdminUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>teacher</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="teacher_id_input">
              <property name="placeholderText">
               <string>Id</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="teacher_name_input">
              <property name="placeholderText">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="teacher_add">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="teacher_table">
            <property name="columnCount">
             <number>2</number>
            </property>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="TeacherRefresh">
            <property name="text">
             <string>RefreshTeacher</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>student</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="student_id_input">
              <property name="placeholderText">
               <string>Id</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="student_name_input">
              <property name="placeholderText">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="student_add">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="student_table">
            <property name="columnCount">
             <number>2</number>
            </property>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StudentRefresh">
            <property name="text">
             <string>RefreshStudent</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>course</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="course_id_input">
              <property name="placeholderText">
               <string>Id</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="course_name_input">
              <property name="placeholderText">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="course_tid_input">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>TeacherId</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="course_add">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="course_table">
            <property name="columnCount">
             <number>3</number>
            </property>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CourseRefresh">
            <property name="text">
             <string>RefreshCourse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Course_Student</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label4">
            <property name="text">
             <string>Course_Student</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="course_student_sid_input">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>StudentId</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="course_student_cid_input">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>CourseId</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="course_student_add">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="course_student_table">
            <property name="columnCount">
             <number>2</number>
            </property>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CourseStudentRefresh">
            <property name="text">
             <string>RefreshCourseStudent</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>work</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label5">
              <property name="text">
               <string>work</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="WorkLibEntry">
              <property name="text">
               <string>使用题库修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="work_table">
            <property name="columnCount">
             <number>4</number>
            </property>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="work_add_button">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="WorkRefresh">
              <property name="text">
               <string>RefreshWork</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>upload</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTableWidget" name="upload_table">
            <property name="columnCount">
             <number>4</number>
            </property>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="upload_add_button">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="UploadRefresh">
              <property name="text">
               <string>RefreshUpload</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="info1">
      <property name="text">
       <string>默认添加的密码为&quot;123456&quot;  双击删除,如果不能删除说明外键限制不满足  表格中的项选择后可以进行筛选  表格中的项选择后可以进行自动填充</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="info2">
      <property name="text">
       <string>work和upload表格双击为编辑 可以通过course表格选择进行筛选</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
